extensions/amp-social-share/0.2/social-share.js

```diff
@@ -52,11 +52,15 @@ export function SocialShare(props) {
       throw new Error('Clicked before href is set.');
     }
     if (startsWith(href, 'navigator-share:')) {
-      if (navigator.share !== undefined) {
```

Comment:
This condition was incorrect (it was copied from a previous `devAssert`, which is inverted.)